incremental-font-transfer/src/patchmap.rs

```diff
@@ -676,19 +818,31 @@ impl PatchUri {
         source_table: IftTableTag,
         application_flag_bit_index: usize,
         encoding: PatchEncoding,
+        intersection_info: IntersectionInfo,
     ) -> PatchUri {
         PatchUri {
             template: uri_template.to_string(),
             id: PatchId::Numeric(entry_index),
             source_table,
             application_flag_bit_index,
             encoding,
+            intersection_info,
+        }
+    }
+}
+
+impl IntersectionInfo {
+    fn from(value: SubsetDefinition, order: usize) -> Self {
```

Comment:
I personally wouldn't use `from` as a fn name (outside of the `From` trait) and I would just call this `new`, if that isn't taken? (or something like `for_subset_definition`, if it is)